README.md

```diff
@@ -151,17 +172,106 @@ tailscale up --advertise-routes="10.0.0.0/24,192.168.0.0/24"
 ```
 
 ### Can I route traffic from machines on my local network to Tailscale endpoints automatically?
-In theory, yes - however it does require manual changes to your routing rules and these will need
-to be updated if you take advantage of WAN fail-over. This has been discussed in more detail
-[here](https://github.com/SierraSoftworks/tailscale-udm/discussions/51).
 
-*Note that we do not currently include this in `tailscale-udm` due to the risk of breaking conflicts in future.*
+Yes! As of January 30, 2025, [two changes][1][2] to Tailscale have made this
+possible. Much credit goes to @tomvoss, who contributed significant effort to
```

Comment:
I'm going to refuse to merge this until you add yourself to the thank you section @jasonwbarnett 😉. You've done a huge amount of work to get this integrated into Tailscale itself in tailscale/tailscale#10828 and deserve to take some credit for the result.

Reply:
The linked PRs speak for themselves 😄

Reply:
Well hey, if you don't add it, I will 😉